app/services/notifications/remove_by_spammer.rb

```diff
@@ -0,0 +1,25 @@
+# remove notifications created for spammer actions:
+# follow user, create comments, create articles
+module Notifications
+  class RemoveBySpammer
+    def self.call(...)
+      new(...).call
+    end
+
+    def initialize(user)
+      @user = user
+    end
+
+    def call
+      return unless user
+
+      Notification.where(notifiable_type: "Follow", notifiable_id: user.follow_ids).delete_all
```

Comment:
I decided to use ids and not dive into `json_data` field because that can take long time taking into account that `notifications` table is usually large (although we delete old notifications)